code/datums/mutations/radioactive.dm

```diff
@@ -6,7 +6,6 @@
 	time_coeff = 5
 	instability = 5
 	difficulty = 8
-	power = 1
```

Comment:
Could you change this into power_coeff? That’s the correct bariable that was meant to be used